Assets/HotmaxEditorVR/inputManager.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class inputManager : MonoBehaviour
+{
+    // -- controller refs
+    public static Valve.VR.InteractionSystem.Hand hand1;
+    public static Valve.VR.InteractionSystem.Hand hand2;
+    public static SteamVR_TrackedController trackedController1;
+    public static SteamVR_TrackedController trackedController2;
+    private SteamVR_TrackedObject trackedObject2;
+    public static SteamVR_Controller.Device device2;
+
+
+    private stateManager _stateManagerMutatorRef;
+    private bool _selectedObjectIsActive = stateManager.selectedObjectIsActive;
+    private stateManager.editorModes _editorMode = stateManager.editorMode;
+    
+
+
+    private void Awake()
+    {
+        //Match up the hardware index with the "steamVR Hand2"
+        //So the Hand2 is always the starting rightmost hand reletive to the HMD
+        //Not sure why steamVR's select hand option doesn't do this, let me know if you figure it out
+        hand1 = GameObject.Find("Hand1").GetComponent<Valve.VR.InteractionSystem.Hand>();
+        hand2 = GameObject.Find("Hand2").GetComponent<Valve.VR.InteractionSystem.Hand>();
+        trackedController1 = hand1.gameObject.GetComponent<SteamVR_TrackedController>();
+        trackedController2 = hand2.gameObject.GetComponent<SteamVR_TrackedController>();
+        trackedObject2 = hand2.GetComponent<SteamVR_TrackedObject>();
+        device2 = SteamVR_Controller.Input((int)trackedObject2.index);
+
+
+        Invoke("setControllerIndecies", 1.2f);
+
+        _stateManagerMutatorRef = GameObject.FindObjectOfType(typeof(stateManager)) as stateManager;
+        stateManager.selectedObjectIsActiveEvent += updateSelectedObjectIsActive;
+        stateManager.editorModeEvent += updateEditorMode;
+    }
+
+    protected virtual void OnApplicationQuit()
+    {
+        stateManager.selectedObjectIsActiveEvent -= updateSelectedObjectIsActive;
+        stateManager.editorModeEvent -= updateEditorMode;
+
+    }
+
+    void updateSelectedObjectIsActive(bool value)
+    {
+        _selectedObjectIsActive = value;
+    }
+
+    void updateEditorMode(stateManager.editorModes value)
```

Comment:
remove this and the listeners if we dont need them right now

Reply:
got it!